testing/_py/test_local.py

```diff
@@ -747,7 +747,7 @@ def test_setmtime(self):
             name = tempfile.mktemp()
             open(name, "w").close()
         try:
-            mtime = int(time.time()) - 100
+            mtime = int(_pytest.timing.time()) - 100
```

Comment:
This particular test needs to be excluded from timing control as it tests fs interaction